src/Tribe/Tickets_Handler.php

```diff
@@ -201,10 +201,16 @@ public function update_shared_tickets_capacity( $meta_id, $object_id, $meta_key,
 			}
 
 			$totals = $this->get_ticket_totals( $ticket );
+			$completes[] = $complete = $totals['pending'] + $totals['sold'];
 
-			$stock = $event_capacity - $totals['pending'] - $totals['sold'];
+			$stock = $event_capacity - $complete;
 			update_post_meta( $ticket, '_stock', $stock );
 		}
+
+		// Make sure we are updating the Global Stock when we update it's capacity
+		$shared_stock = new Tribe__Tickets__Global_Stock( $object_id );
+		$shared_stock_level = $event_capacity - array_sum( $completes );
```

Comment:
Not sure if this would happen in practice, but it _looks_ like there is potential for the foreach loop not to run (ie, if `$tickets` is empty) ... if that is indeed possible we might create some noise and unexpected results here, as `$completes` won't be defined

Reply:
Ohh good catch. Will add a base definition.
But yes it wouldn't happen.